From ebf64f6a531e4b279e8995af3a0ae229a6169e78 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Thu, 17 Jan 2019 16:50:24 +0100
Subject: [PATCH] Merge pull request #405 in SW_UGW/linux from
 feature/PONRTSYS-2654-remove-falconmx to master

* commit '59561a7026db87bc622c1768337a77ee0aaeca2b':
  PONRTSYS-2654: remove falconmx occurrences from qos and BM drivers (Also dts was changed)
---
 drivers/net/ethernet/lantiq/ppv4/bm/pp_bm_drv.c     | 2 +-
 drivers/net/ethernet/lantiq/ppv4/qos/pp_qos_linux.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/ppv4/bm/pp_bm_drv.c b/drivers/net/ethernet/lantiq/ppv4/bm/pp_bm_drv.c
index 535ad8a9d23c..2742b8b721fb 100644
--- a/drivers/net/ethernet/lantiq/ppv4/bm/pp_bm_drv.c
+++ b/drivers/net/ethernet/lantiq/ppv4/bm/pp_bm_drv.c
@@ -64,7 +64,7 @@ static int buffer_manager_remove(struct platform_device *pdev);
 #define RC_SUCCESS	(0)
 
 static const struct of_device_id bm_match[] = {
-	{ .compatible = "intel,falconmx-bm" },
+	{ .compatible = "intel,prx300-bm" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, bm_match);
diff --git a/drivers/net/ethernet/lantiq/ppv4/qos/pp_qos_linux.c b/drivers/net/ethernet/lantiq/ppv4/qos/pp_qos_linux.c
index 7d8ea8f812cd..c0066aae3f3c 100644
--- a/drivers/net/ethernet/lantiq/ppv4/qos/pp_qos_linux.c
+++ b/drivers/net/ethernet/lantiq/ppv4/qos/pp_qos_linux.c
@@ -97,7 +97,7 @@ int load_firmware(struct pp_qos_dev *qdev, const char *name)
 }
 
 static const struct of_device_id pp_qos_match[] = {
-	{ .compatible = "intel,falconmx-ppv4qos" },
+	{ .compatible = "intel,prx300-ppv4qos" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, pp_qos_match);
